### ISSUE TITLE
Why cudnn make loss change？

### ISSUE THREAD

**Issue by XiaoYee** [pytorch/pytorch#4905]
when i use cudnn in pytorch，namely set torch.backends.cudnn.enabled = True，then my train loss will be different every time when i begin to train.

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for this question
